docs/examples/example.obo

```diff
@@ -370,25 +371,25 @@ xref: http://www.snomedbrowser.com/Codes/Details/362584002
 xref: MA:0000118
 xref: MAT:0000494
 xref: MESH:A06.407.071.140
-xref: UMLS:C0001613
+xref: UMLS:C0001613 {source="ncithesaurus:Adrenal_Cortex"}
 xref: VHOG:0001481
 xref: XAO:0000165
 is_a: UBERON:0001851 ! cortex
 intersection_of: UBERON:0001851 ! cortex
 intersection_of: part_of UBERON:0002369 ! adrenal gland
-property_value: depicted:by http://upload.wikimedia.org/wikipedia/commons/5/5c/Gray1185.png
-property_value: depicted:by http://upload.wikimedia.org/wikipedia/commons/thumb/5/5c/Gray1185.png/200px-Gray1185.png
-property_value: homology_notes "All craniates have groups of cells homologous to the mammalian adrenocortical and chromaffin tissues (medulla), but they are scattered in and near the kidneys in fishes. (...) The cortical and chromaffin tissues come together to form adrenal glands in tetrapods.[well established][VHOG]" xsd:string
+property_value: depicted:by "http://upload.wikimedia.org/wikipedia/commons/5/5c/Gray1185.png"
```

Comment:
Interesting that literals are now wrapped in ""

@hkir-dev is this backwards compatible, i.e. a non-wrapped IRI (OBO format) can still be parsed by the new owlapi?

Reply:
yes, it is backward compatible. Not quoted and quoted IRIs produced the same ontology.